book/internal-services.md

```diff
@@ -0,0 +1,83 @@
+# Internal Services
+
+## Overview
```

Comment:
Nit: Is capitalization in this section because they are "keywords"? It feels a little odd since they are not necessarily proper nouns.

Reply:
Fixed